sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -880,7 +885,7 @@ case class IntegralDivide(
     DecimalType.bounded(if (intDig == 0) 1 else intDig, 0)
   }
 
-  override def sqlOperator: String = "div"
+  override def sqlOperator: String = "DIV"
```

Comment:
apology I joined this party late. Seems like doing uppercase also needs a lot of changes? How big is it to make all lowercased? Otherwise, I tend to think that it might not be so worthwhile fixing.

Reply:
If we change it to lowercase, there will be approximately `55` file changes, as shown in:

The original intention of this PR is to maintain consistency, whether it is lowercase or uppercase (Currently, it seems that using uppercase letters to display mathematical function names is more in line with common practice). Two confusing cases are described in the PR.
- Inconsistent case of alias
   <img width="560" alt="image" src="https://github.com/apache/spark/assets/15246973/5e073f37-0767-4922-b62a-b3a54fb9fb80">

- The number of function parameters also leads to inconsistent capitalization